### ISSUE TITLE
Deploying static site with WPML sometimes throws exception

### ISSUE THREAD

**Issue by ALobker** [Simply-Static/simply-static#266]
We are using WPML to translate our website into 3 different languages. When running the generation, sometimes the activity log shows an exception. This is not always the case, so we are not sure what causes it.

Simply Static version: Pro 1.6.1

Steps to manually fix it:
- Login with ftp to server with wordpress page.
- Go to folder /wp-content/uploads/simply-static/temp-files/
- Start the script again from the wordpress backend
- With ftp, go into the simply-static-1-[number] folder
- Quickly create the language folders (/de, /es, /fr)
- Wait for the deployment process to finish

Stack trace here:
```
[2024-10-22 19:33:48] Setting up
[2024-10-22 19:34:20] Fetched 288 of 288 pages/files
[2024-10-22 19:36:07] Minified all pages/files
[2024-10-22 19:36:07] Error: (1) Uncaught Exception: /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/uploads/simply-static/temp-files/simply-static-1-1729618427/es/wp-content|Could not create the destination directory. in /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/plugins/simply-static-pro/src/integrations/wpml/class-ssp-wpml-task.php:176
Stack trace:
#0 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/plugins/simply-static-pro/src/integrations/wpml/class-ssp-wpml-task.php(65): simply_static_pro\WPML_Task->copy_langs()
#1 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/plugins/simply-static/src/class-ss-archive-creation-job.php(154): simply_static_pro\WPML_Task->perform()
#2 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/plugins/simply-static/vendor/a5hleyrich/wp-background-processing/classes/wp-background-process.php(531): Simply_Static\Archive_Creation_Job->task()
#3 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/plugins/simply-static/vendor/a5hleyrich/wp-background-processing/classes/wp-background-process.php(350): WP_Background_Process->handle()
#4 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-includes/class-wp-hook.php(324): WP_Background_Process->maybe_handle()
#5 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters()
#6 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-includes/plugin.php(517): WP_Hook->do_action()
#7 /home/[hidden]/domains/wp.[hidden].com/public_html/wp-admin/admin-ajax.php(192): do_action()
#8 {main}
  thrown in /home/[hidden]/domains/wp.[hidden].com/public_html/wp-content/plugins/simply-static-pro/src/integrations/wpml/class-ssp-wpml-task.php on line 176
```

**Comment by patrickposner** [comment]
This should be solved with the next update - we pushed some improvements for the WPML task to speed things up and ensure the tasks continue in case of any interruption (timeout, connection loss, etc..)
